### ISSUE TITLE
Deploy Default mock assets and safeguarding wrapped assets

### ISSUE THREAD

**Issue by Primata** [movementlabsxyz/movement#203]
## **Is your feature request related to a problem? Please describe.**
As a network we must deploy mock tokens that should be used by all protocols in our testnet and provide a way to get access to those assets.

Common assets to be supported are USDC, USDT, BTC and ETH. USDC and USDT should be treated as assets for stableswapping but there is no way for us to enforce it.

## **Describe the solution you'd like**
* USDC, USDT, WBTC (aka. e.WBTC), should be mintable by calling a mint function on Aptos, MEVM and Sui.
* MOVETH (aka. ETH) is provided by using our native bridge on Aptos. MOVETH should be mintable by calling a mint function on MEVM and Sui.
* MOVE is our native asset, 0x1::aptos_coin::AptosCoin, 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE and 0x2::sui::SUI.
* WMOVE will be deployed on MEVM using WETH10. it's not common practice to wrap native move assets on aptos or sui.

**Mintable** assets will be available on explorers as a Modal whenever testnet is selected.

Aptos has the highest priority.

## **Describe alternatives you've considered**
Just allowing any asset to be mintable but maintaining a central authority of what is the default asset is a must.

**Comment by andyjsbell** [comment]
Could we cut back the scope to just our testnet Suzuka?

**Comment by 0xPrimata** [comment]
> Could we cut back the scope to just our testnet Suzuka?

So only Aptos assets?

**Comment by andygolay** [comment]
If MOVE will be using `0x1::aptos_coin::AptosCoin`

and MovETH will be using FA, are there any potential conflicts, on testnet?

How important is it that these coins are interoperable with staked ETH?

If it is important, then should we consider the FA model?

**Comment by 0xPrimata** [comment]
> If MOVE will be using `0x1::aptos_coin::AptosCoin`
> 
> and MovETH will be using FA, are there any potential conflicts, on testnet?
Yes they shouldn't have issues, FA is an extension of coin for better functionality.
> How important is it that these coins are interoperable with staked ETH?
Staked ETH? Where is the staked ETH coming from?
> If it is important, then should we consider the FA model, or do we have solutions for Coin <> FA interop?
Believe it's interop.

**Comment by andyjsbell** [comment]
Cool. Makes sense to keep the scope tight and just for Suzuka @0xPrimata

**Comment by 0xmovses** [comment]
MOVETH is only mintable upon depositing in the bridge contract on an Ethereum network (testnet or otherwise). It's impossible to have a faucet for Moveth as it represent locked wrapped WETH on the eth network. Maybe we should remove it from the list?

**Comment by 0xmovses** [comment]
Do you have the move module code for USDC and USDT? With WBTC, again this implies wrapping that asset first, so some BTC on the bitcoin network. And then bridging that across.

**Comment by 0xPrimata** [comment]
@0xmovses on WBTC, just went with the standard nomeclature for L2s that do not use BTC or ETH natively. It's referred as WBTC or WETH when they are ERC20s. Some DEXs might refer to them as BTC or ETH.
